apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/components/AssessmentSummary.tsx

```diff
@@ -0,0 +1,254 @@
+  if (
+    educationAssessmentResult[0] &&
+    educationAssessmentResult[0].assessmentDecision ===
+      AssessmentDecision.Successful
+  ) {
+    educationAssessmentResultDecision = AssessmentDecision.Successful;
+  } else if (
+    educationAssessmentResult[0] &&
+    educationAssessmentResult[0].assessmentDecision ===
+      AssessmentDecision.Unsuccessful
+  ) {
+    educationAssessmentResultDecision = AssessmentDecision.Unsuccessful;
+  }
+
+  // for each skill, given the assessment results array, compute the results and generate table data
+  const essentialSkillsTableData = essentialSkills.map((skill) => {
+    const essentialSkillCalculated = skillAssessmentResultCalculator(
+      skill,
+      assessmentResults,
+    );
+    return (
+      <tr key={skill.id}>
+        <td data-h2-padding="base(x.25 0 x.25 x1)">
+          {getLocalizedName(skill.name, intl)}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {essentialSkillCalculated.successful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {essentialSkillCalculated.unsuccessful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {essentialSkillCalculated.hold}
+        </td>
+      </tr>
+    );
+  });
+  const nonessentialSkillsTableData = nonessentialSkills.map((skill) => {
+    const nonessentialSkillCalculated = skillAssessmentResultCalculator(
+      skill,
+      assessmentResults,
+    );
+    return (
+      <tr key={skill.id}>
+        <td data-h2-padding="base(x.25 0 x.25 x1)">
+          {getLocalizedName(skill.name, intl)}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {nonessentialSkillCalculated.successful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {nonessentialSkillCalculated.unsuccessful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {nonessentialSkillCalculated.hold}
+        </td>
+      </tr>
+    );
+  });
+
+  return (
+    <>
+      <table data-h2-background="base(background)">
+        <thead>
+          <tr
+            data-h2-border-bottom="base(3px solid black.20)"
+            data-h2-margin-bottom="base(x.5)"
+          >
+            <th
+              scope="col"
+              data-h2-padding="base(x.25 0 x.25 x.5)"
+              data-h2-display="base(flex)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Essential criteria",
+                description: "Essential criteria heading",
+                id: "Kp3Bqu",
+              })}
+            </th>
+            <th scope="col">
+              <CheckIcon
+                data-h2-width="base(x1)"
+                data-h2-display="base(flex)"
+                data-h2-vertical-align="base(bottom)"
+                data-h2-margin="base(0, x.5, 0, 0)"
+                data-h2-padding="base(x.25 0)"
+                data-h2-color="base(success)"
+              />
```

Comment:
All of these icons will need an `aria-label`.

Reply:
Labels added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/2a6a05420a146d090167d6291d72d4dbfd25383d

Reply:
If we are able to drop the "assessment" portion of these, we already have the status text translated so we can keep some consistency in how they are referenced.

Reply:
Sure, reused in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/4c9f36480cff2ed527aff5e658aaac1a335c0350